driver-sync/src/main/com/mongodb/client/internal/MongoClientDelegate.java

```diff
@@ -71,12 +74,28 @@ public OperationExecutor getOperationExecutor() {
     }
 
     @Nullable
-    public ClientSession createClientSession(final ClientSessionOptions options) {
+    public ClientSession createClientSession(final ClientSessionOptions options, final ReadConcern readConcern,
+                                             final WriteConcern writeConcern) {
+        notNull("readConcern", readConcern);
+        notNull("writeConcern", writeConcern);
+        ClientSessionOptions.Builder newOptionsBuilder = ClientSessionOptions.builder();
```

Comment:
More ugliness so that we can replace the supplied default options with ones that apply the default options from the mongo client.